### PR TITLE
4917 - Lightened the colors for disabled breadcrumbs

### LINKED ISSUE #4917: Breadcrumb: Disabled item color contrast in Uplift
**Describe the bug**
elements that have the disabled attribute does not have enough contrast to their text from enabled ones

**To Reproduce**

1. Go to https://4380-beta0-enterprise.demo.design.infor.com/components/breadcrumb/example-add-remove.html
2. Add new Breadcrumbs (disabled and enabled)
3. Observe and compare the text color of the breadcrumb to enabled ones

**Expected behavior**
The disabled breadcrumb should provide enough contrast to differentiate it from the enabled one.

**Version**
- ids-enterprise: v4.38.0-beta0

**Screenshots**
image

**Platform**
All major browsers

**Additional context**

### PR DESCRIPTION
**Pull request by chrisakinrinade**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Disabled breadcrumbs currently have no visual distinction from enabled breadcrumbs - this solves that by making them lighter.

**Related github/jira issue (required)**:
Fixes #4917 

**Steps necessary to review your pull request (required)**:
1. Visit http://localhost:4000/components/breadcrumb/example-add-remove.html
2. Click Add button
3. Click disabled check box
4. Click Add button again
Observe the difference in coloring between the two breadcrumbs
image

**Included in this Pull Request**:
- [x] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

File: + [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Merge branch 'master' of https://github.com/infor-design/enterprise
```
Comment (tmcconechy) [inline +:1]: Whats this file? Maybe added by mistake? (its wierd it has no name)

**Review by tmcconechy — CHANGES_REQUESTED** [review]
A couple comments:

1. whats that empty file with PR info in it
2. whats teh difference between http://localhost:4000/components/breadcrumb/example-add-remove-disabled.html and http://localhost:4000/components/breadcrumb/example-add-remove.html (did we need two identical? examples)
3. in some themes its still not disabled looking enough (bring the color/down/ up a bit?) in themes:
 - classic - dark
 - new - dark
 - classic - contrast
 - new - contrast

File: test/components/breadcrumb/breadcrumb.e2e-spec.js [hunk 2]
```diff
@@ -105,3 +106,20 @@ describe('Breadcrumb automation tests', () => {
...
   });
 });
+
+describe('Breadcrumb should be accessible with no WCAG 2AA violations', () => {
+  if (!utils.isIE()) {
+    const themes = ['new', 'classsic'];
+    const modes = ['light', 'dark', 'contrast'];
+    themes.forEach((theme) => {
+      modes.forEach(async (mode) => {
+        await utils.setPage(`/components/breadcrumb/example-add-remove-disabled?theme=${theme}&mode=${mode}`);
```
Comment (EdwardCoyle) [inline test/components/breadcrumb/breadcrumb.e2e-spec.js:116]: @tmcconechy I'd mentioned to @chrisakinrinade that we didn't have any sample pages that demonstrated a disabled Breadcrumb item that we could easily use for visual regression/aXe testing, hence the duplicate page (used here).

Chris, you probably don't need to have the entire add/remove functionality in the new breadcrumb example page.  Just placing the breadcrumb with the disabled item should be sufficient.  👍

Comment (EdwardCoyle) [reply test/components/breadcrumb/breadcrumb.e2e-spec.js:116]: Thanks for the added test BTW

**Review by EdwardCoyle — COMMENTED** [review]
See below:

File: test/components/breadcrumb/breadcrumb.e2e-spec.js (hunk 2 above)
Comment (tmcconechy) [reply test/components/breadcrumb/breadcrumb.e2e-spec.js:116]: Yes thats what i also said to @chrisakinrinade in chat. could make an example-disabled page that is just showing a few disabled tags based on example-index so we can make a test without clicking all the buttons

**Comment by tmcconechy** [comment]
Looks like you might need to update the test / test link or remove this test https://github.com/infor-design/enterprise/pull/4960/checks?check_run_id=2171627316#step:10:3351

Is the rest ready to review?

**Comment by chrisakinrinade** [comment]
Ok, I removed one test and tweaked the others. Ready to review! @tmcconechy

**Review by tmcconechy — DISMISSED** [review]
Looks good to me now. Fix is correct and tests look good.

**Review by tmcconechy — APPROVED** [review]

**Review by EdwardCoyle — APPROVED** [review]
Looks great!  :banana:

**Pull request merged**